webhook/psbinding/reconciler.go

```diff
@@ -229,6 +235,46 @@ func (r *BaseReconciler) RemoveFinalizer(ctx context.Context, fb kmeta.Accessor)
 	return err
 }
 
+func (r *BaseReconciler) labelNamespace(ctx context.Context, subject tracker.Reference) error {
+
+	// Determine the GroupVersionResource of the subject reference
+	gvr := schema.GroupVersionResource{
+		Group:    "",
+		Version:  "v1",
+		Resource: "namespaces",
+	}
+
+	// Don't remove any existing labels or add ours more than once
+	_, namespaceLister, err := r.Factory.Get(gvr)
+	if err != nil {
+		logging.FromContext(ctx).Infof("Error getting lister for namespaces: '%v': %v", gvr, err)
+		return err
+	}
+	namespaceObject, err := namespaceLister.Get(subject.Namespace)
+	if apierrs.IsNotFound(err) {
+		logging.FromContext(ctx).Infof("")
+		return err
+	} else if err != nil {
+		return err
+	}
+	labels := (namespaceObject.(*duckv1.WithPod)).GetLabels()
+	if labels["bindings.knative.dev/exclude"] != "" || labels["bindings.knative.dev/include"] != "" {
```

Comment:
wanna use the constants here too:

* https://github.com/knative/pkg/blob/master/apis/duck/const.go#L26
* https://github.com/knative/pkg/blob/master/apis/duck/const.go#L32